src/backend/storeManagers/sideload/library.ts

```diff
@@ -73,7 +68,7 @@ export function addNewApp({
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 
-export function installState(appName: string, state: boolean) {
+export function installState() {
```

Comment:
we don't need the arguments in this functions because we don't use them

removing to avoid importing types we don't need